resources/views/partials/content-single-pcc-story.blade.php

```diff
@@ -0,0 +1,39 @@
+@php $org = get_post_meta (get_the_ID(), 'pcc_story_organization', true) @endphp
```

Comment:
It's best practice to pass data to the template by creating a controller method. In this case, adding a method called `storyOrg` in the SinglePccStory controller would populate a variable called `$story_org`:

```php
public function storyOrg()
{
  return get_post_meta(get_the_ID(), 'pcc_story_organization', true);
}

Alternatively, you could create a static method in the trait (see previous comment) which would handle this for both the Page and Single views but would need to be called (`Page::storyOrg()` etc).